docker/haveno/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build haveno-seednode and haveno-daemon. Managed by docker-compose.
+
+FROM openjdk:11
+
+RUN set -ex && \
+    apt update && \
+    apt --no-install-recommends --yes install \
+        make \
+        git
+
+RUN set -ex && adduser --system --group --disabled-password haveno && \
+    mkdir -p /home/haveno && \
+    chown -R haveno:haveno /home/haveno
+
+USER haveno
+
+RUN set -ex && git clone -b docker https://github.com/napoly/haveno.git /home/haveno/haveno
```

Comment:
change to:
`RUN set -ex && git clone https://github.com/haveno-dex/haveno.git /home/haveno/haveno`
before merging to master